_posts/2020-07-08-Book-Five-Dysfunctions.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+authors: [peter_dekinder]
+title: 'Book Review - The Five Dysfunctions of a Team'
```

Comment:
I would not call it just 'book review'. 
I think you also analysed, formulated your own Insights and linked it to other sources while doing that